emscripten.py

```diff
@@ -1493,14 +1493,18 @@ def create_named_globals(metadata):
 def create_runtime_funcs(exports):
   if shared.Settings.ONLY_MY_CODE:
     return []
-  return ['''
+
+  stack_check = '  if ((STACKTOP|0) >= (STACK_MAX|0)) abortStackOverflow(size|0);\n'
+  enable_stack_check = shared.Settings.ASSERTIONS or shared.Settings.STACK_OVERFLOW_CHECK >= 2
+
+  rtn = ['''
```

Comment:
`rtn` feels like a too-temporary name to me for how long this function is. How about `funcs` or `runtime_funcs`?